cfme/tests/cloud_infra_common/test_genealogy.py

```diff
@@ -22,6 +25,30 @@
         scope='module'),
     test_requirements.genealogy
 ]
+
+
+@pytest.fixture("function")
+def clone_vm_name():
+    clone_vm_name = fauxfactory.gen_alphanumeric(18, start="test_genealogy_")
+    return clone_vm_name
```

Comment:
Rather than this fixture, you can use the `random_vm_name` function https://github.com/ManageIQ/integration_tests/blob/77f5365ff733b6cadfb47e7cba60bae647088428/cfme/utils/generators.py#L6